tests/pages/firefox/os/tv.py

```diff
@@ -0,0 +1,72 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage, FirefoxBasePageRegion
+
+
+class TVPage(FirefoxBasePage):
+
+    _url = '{base_url}/{locale}/firefox/os/devices/tv'
+
+    _next_button_locator = (By.CSS_SELECTOR, '.pager-next')
+    _previous_button_locator = (By.CSS_SELECTOR, '.pager-prev')
+    _screens_locator = (By.CSS_SELECTOR, '.pager-page')
+    _thumbnails_locator = (By.CSS_SELECTOR, '.pager-tabs li')
+
+    @property
+    def screens(self):
+        return [Screens(self.base_url, self.selenium, root=el) for el in
+                self.find_elements(self._screens_locator)]
+
+    @property
+    def thumbnails(self):
+        return [Thumbnails(self.base_url, self.selenium, root=el) for el in
+                self.find_elements(self._thumbnails_locator)]
+
+    def show_next_screen(self):
+        screen = next(s for s in self.screens if s.is_displayed)
+        self.find_element(self._next_button_locator).click()
+        self.wait.until(lambda s: not screen.is_displayed)
+
+    def show_previous_screen(self):
+        screen = next(s for s in self.screens if s.is_displayed)
+        self.find_element(self._previous_button_locator).click()
+        self.wait.until(lambda s: not screen.is_displayed)
+
+    def click_thumbnail(self, i):
```

Comment:
I wonder if we should add a comment that this method is necessary because the thumbnail region does not have a reference to the page. Perhaps not, but it might help us to remember to clean this up if we add that ability.